src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FruitProcessing;
+import core.basesyntax.service.impl.FruitProcessingImpl;
+
+public class Main {
+    public static void main(String[] args) {
+        FruitProcessing fruitProcessing = new FruitProcessingImpl();
+        fruitProcessing.fruitProcessing("inputData.csv");
```

Comment:
Now you have single service that doing enormous amount of tasks and for unprepared person it's vary hard to understand the flow.

In you main method you should do following things:
1. Some new service (e.g. FileReader) will read content of input file (probably to List<String>)
2. Some other service (e.g. TransactionParser) will parse strings to some mode (e.g. FruitTransaction)
3. One more service will take these transations and change state of some DB analog (Map<String, Integer> in separate class Storage is easiest option to go with, where key is fruit name and value is quantity) processing them one-by-one.
4. Service from point 3 will require strategy, which will provide proper TransactionHandler depending on Transaction type.
5. And handlers will contain method to change storage state (each for operation it represents)
6. After all of that there will be additional service that generates report text based on data in storage.
7. And then some FileWriter service will take this report and write it to output file